components/Starknet/modules/architecture_and_concepts/pages/Network_Architecture/transactions.adoc

```diff
@@ -377,6 +377,7 @@ Where:
 * `deploy_account` is a constant prefix string, encoded in ASCII.
 include::partial$snippet_transaction_params.adoc[]
 * `class_hash` is the hash of the xref:architecture_and_concepts:Smart_Contracts/contract-classes.adoc[contract class]. See xref:architecture_and_concepts:Smart_Contracts/class-hash.adoc[Class Hash] for details about how the hash is computed.
+* `contract_address` is calculated as described xref:architecture_and_concepts:Smart_Contracts/contract-address.adoc[here].
```

Comment:
@odednaor So is this address _not_ the sender's address, but rather the address of the new account that is being deployed?

Reply:
Yes. A deploy account transaction doesn't have a `sender_address` field.